platform/30-kyverno-setup.sh

```diff
@@ -6,6 +6,25 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhyQCx0E9wQWSFI9ULGwy3BuRklnt
 IqozONbbdbqz11hlRJy9c7SG+hdcFl9jE9uE/dwtuwU2MqU9T/cN0YkWww==
 -----END PUBLIC KEY-----"
 
+REPO="ttl.sh/*"
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+KYVERNO_INSTALL_DIR=${GIT_ROOT}/platform/vendor/kyverno/release
+KYVERNO_RESOURCE_DIR=${GIT_ROOT}/resources/kyverno/admission-control-policy
+
+# Define variables.
+C_GREEN='\033[32m'
+C_YELLOW='\033[33m'
+C_RED='\033[31m'
+C_RESET_ALL='\033[0m'
+
+# Wait until pods are ready.
+# $1: namespace, $2: app label
+wait_for_pods () {
+  echo -e "${C_YELLOW}Waiting: $2 pods in $1...${C_RESET_ALL}"
+  kubectl wait --timeout=5m --for=condition=ready pods -l app=$2 -n $1
+}
+
```

Comment:
I think this part is duplicated below.

Reply:
Yea not sure what happened with the upstream merger. I will need to squash commits so I'll get it all sorted.